pkg/registry/errors.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-func newImageAccessError(ref string, err error) error {
+func NewImageAccessError(ref string, err error) error {
```

Comment:
Is this intentionally exported?

Reply:
yes, so the source.Uploader could use it too.. it was previously duplicated in the SourceUploader

Reply:
marking as resolved. addressed in commit https://github.com/vmware-tanzu/kpack-cli/commit/be211733ecaa33231703f7b9d2938b0b71bee3a3
please reopen if not ok